ui-tests/src/test/resources/features/integrations/database-to-database-with-extension.feature

```diff
@@ -21,7 +21,7 @@ Feature: Upload tech extension and add it to integration
     When Camilla clicks on the "Import Extension" button
     Then she is presented with the Syndesis page "Import Extension"
 
-    When Camilla upload extension
+    When Camilla upload extension "extension_name"
```

Comment:
The name should be ` When Camilla upload extension "syndesis-extensions-1.0.0-SNAPSHOT"`